### ISSUE TITLE
code comment showing on main page

### ISSUE THREAD

**Issue by riftrid3r** [riftrid3r/wpjen#22]
#21 
@JLAnichowski , this comment is showing up in the top left corner of index. The comment should look like this:

//comment text

here is a link to the documentation for comments in php.
http://php.net/manual/en/language.basic-syntax.comments.php

**Comment by JLAnichowski** [comment]
so should I go in and change it to that?

**Comment by riftrid3r** [comment]
yeah go for it! I am at my desk right now if you have any questions.
